iam_openapi_spec.yml

```diff
@@ -2878,6 +2916,15 @@ components:
             action: hrn:XlK5h64gND::invoice$view
             effect: allow
 
+    CreatePolicyFromTemplateRequest:
+      summary: sample create policy from template request
+      value:
+        name: Product Manager Policy
```

Comment:
Are we allowing spaces in resource names today? We must add charset restrictions

Reply:
No and this restriction is added in requestModelValidator